AAEmu.Game/Models/Game/Skills/Effects/SpecialEffects/AddBreath.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Linq;
+using AAEmu.Game.Core.Managers;
+using AAEmu.Game.Core.Managers.World;
+using AAEmu.Game.Models.Game.Char;
+using AAEmu.Game.Models.Game.Items;
+using AAEmu.Game.Models.Game.Units;
+using NLog;
+
+namespace AAEmu.Game.Models.Game.Skills.Effects.SpecialEffects
+{
+    public class AddBreath : SpecialEffectAction
+    {
+        private static Logger _log = LogManager.GetCurrentClassLogger();
+
+        public override void Execute(Unit caster,
+            SkillCaster casterObj,
+            BaseUnit target,
+            SkillCastTarget targetObj,
+            CastAction castObj,
+            Skill skill,
+            SkillObject skillObject,
+            DateTime time,
+            int value1,
+            int value2,
+            int value3,
+            int value4)
+        {
+            if (caster is Character character && character.IsUnderWater)
+                character.Breath = character.MaxBreath;
```

Comment:
This should not set to MaxBreath but add value1, which appears to be breath time to add in msec 

so

`character.Breath = Math.Min(character.LungCapacity, character.Breath + value1)`